experimenter/experimenter/features/tests/test.yaml

```diff
@@ -0,0 +1,9 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+---
+fenix:
+  repo: \@mozilla-mobile/firefox-android
```

Comment:
I think we should just not include the @ in these and add the @ to the CLI invocations etc when necessary.

Reply:
agreed, let's keep it for now until we can get that into the CLI. I confirmed with James.